assets/js/blocks/product-categories/index.js

```diff
@@ -83,6 +83,26 @@ registerBlockType( 'woocommerce/product-categories', {
 		},
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				blocks: [ 'core/legacy-widget' ],
+				// We can't transform if raw instance isn't shown in the REST API.
+				isMatch: ( { idBase, instance } ) =>
+					idBase === 'woocommerce_product_categories' &&
+					!! instance?.raw,
+				transform: ( { instance } ) =>
+					createBlock( 'woocommerce/product-categories', {
+						hasCount: instance.raw.count,
```

Comment:
I see count and some other boolean properties come back as integers. Should we convert to boolean here? e.g. `hasCount: !! instance.raw.count`

Reply:
Sure :)